packages/react/src/utilities/positioning/positioning.test.ts

```diff
@@ -287,6 +288,14 @@ describe('Callout Positioning', () => {
     expect(finalizedPosition.elementPosition.bottom).toBeDefined();
     expect(finalizedPosition.elementPosition.right).toBeUndefined();
 
+    // With bounds flush against the edge of the elementRectangle, change the edge such that the elementRectangle
+    // will grow correctly but will not visually change
+    // In this case, that's bottom (opposite of target) and right (flush against the bounds)
+    finalizedPosition = __positioningTestPackage._finalizePositionData(pos, host as any, flushBounds, false, true);
+    expect(finalizedPosition.elementPosition.left).toBeUndefined();
+    expect(finalizedPosition.elementPosition.bottom).toBeDefined();
+    expect(finalizedPosition.elementPosition.right).toBeDefined();
+
```

Comment:
Is it possible to add a new test for the fix, and preserve this test to make sure we're not breaking any other existing behavior? Or is this test failing because of the fix?

Reply:
This change breaks this test, but I could always add another one specific to this behaviour